v2/tools/generator/internal/test/resource.go

```diff
@@ -65,3 +78,9 @@ func CreateSimpleResource(
 	status := CreateStatus(pkg, name)
 	return CreateResource(pkg, name, spec, status)
 }
+
+func CreateTypeName(
```

Comment:
This method doesn't seem to add a ton, since it's just calling `astmodel.MakeTypeName`? Why not just call `astmodel.MakeTypeName` wherever you're using this and delete this method?

Reply:
Removed this method, thought we have to have helper methods in the test package. Have made the changed to use `astmodel.MakeTypeName` .